src/Domain/Infra/Doctrine/MappingConfig.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\Domain\Infra\Doctrine;
+
+/**
+ * @author Pascal Wacker <hello@pascalwacker.ch>
+ *
+ * @internal
+ */
+final class MappingConfig
```

Comment:
we should make it a simple DTO / throwaway object

```php
final class MappingConfig
{
  public $mappingFiles;
  public $mappingDir;
  public $keyMaxLength;
}
```

only `mappinFiles` are required in the constructor, the rest is nullable for now (if null we do nothing)